README.md

```diff
@@ -96,18 +96,18 @@ Edit `mempool-config.json` to add your Bitcoin Core node RPC credentials:
   "MEMPOOL": {
     "NETWORK": "mainnet",
     "BACKEND": "electrum",
-    "HTTP_PORT": 8999,
-    "API_URL_PREFIX": "/api/v1/",
```

Comment:
this is still needed

Reply:
this is not a full configuration file, it's just suggesting what to change from copying the full one